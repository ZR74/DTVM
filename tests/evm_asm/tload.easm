PUSH1 0x2e
PUSH1 0x00
TSTORE
PUSH1 0x00
TLOAD
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN