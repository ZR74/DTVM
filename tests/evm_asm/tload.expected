status: success
error_code: 0
stack: []
memory: '000000000000000000000000000000000000000000000000000000000000002E'
storage: {}
transient_storage: {}
return: '000000000000000000000000000000000000000000000000000000000000002E'
events: []
